policyengine_us/variables/gov/states/md/tax/income/credits/ctc/md_ctc.py

```diff
@@ -24,8 +25,28 @@ def formula_2020(tax_unit, period, parameters):
         meets_age_limit = person("age", period) < age_limit
         eligible = dependent & meets_age_limit
         eligible_children = tax_unit.sum(eligible)
-        md_ctc = eligible_children * p.amount
+
+        # Calculate base credit amount
+        base_credit = eligible_children * p.amount
+
+        # Apply federal CTC reduction if applicable
         if p.reduced_by_federal_credit:
             federal_ctc = tax_unit("ctc", period)
-            return max_(md_ctc - federal_ctc, 0)
-        return md_ctc
+            base_credit = max_(base_credit - federal_ctc, 0)
+
+        # Starting in 2025, apply income-based phase-out instead of hard cutoff
+        if period.start.year >= 2025:
```

Comment:
We should make the 2025 parametric but I agree with @DTrim99 that a parallel structure seems cleaner